client/packages/system/src/Stock/Components/StockLineEditModal.tsx

```diff
@@ -117,8 +126,7 @@ export const StockLineEditModal: FC<StockLineEditModalProps> = ({
 
   return (
     <Modal
-      width={700}
-      height={575}
+      sx={{ maxWidth: 'unset', minWidth: 700, minHeight: 575 }}
```

Comment:
This makes the modal more flexible for the different requirements of the different tabs.

Reply:
The jumpy modal feels a bit weird to me, but I think I remember you saying you played around with widening it for all tabs and that messed with the feng shui of the form tabs... and I've seen at least one other modal that changes size on different tabs so I guess all good? at least until we get some more authoritative UX input

Reply:
Yeah, I don't love it, but I found it preferable to having one-size-fits-all which looks decidedly weird in some tabs. Need some UI design at some point I think.